### ISSUE TITLE
How to train on one machine with 4GPUS?

### ISSUE THREAD

**Issue by dk-liang** [IBM/CrossViT#2]
How to train on one machine with 4GPUS?

**Comment by chunfuchen** [comment]
Hi, I won't be able to guarantee the performance when you train with 4 GPUs.

You can simply reduce the number of GPUs and the warmup-epochs, the learning rate is scaled linearly based on your total batch size. (We use 30 epochs for warmup because our batch size is 4,096.)

E.g.
```
python -m torch.distributed.launch --nproc_per_node=4 --use_env main.py --model crossvit_9_dagger_224 --batch-size 256 --data-path /path/to/imagenet --warmup-epochs 5
```

**Comment by chunfuchen** [comment]
closed, please feel free to re-open if you have any other questions
